subdomain-service/scripts/build-all-versions.sh

```
@@ -0,0 +1,126 @@
+  rm -rf "$SUBDOMAIN_SERVICE_DIR/builds/amica_v"*
+  echo ""
+fi
+
+# Track build results
+SUCCESSFUL_BUILDS=()
+FAILED_BUILDS=()
+
+# Function to build a single version
+build_version() {
+  local VERSION_CONFIG=$1
+  local VERSION=$(echo "$VERSION_CONFIG" | cut -d: -f1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)